cli/kubernetes/proxy.go

```diff
@@ -40,15 +39,26 @@ func StartProxy(kubernetesProvider *Provider, mizuPort uint16, mizuNamespace str
 	return server.Serve(l)
 }
 
-func GetMizuCollectorProxiedHostAndPath(mizuPort uint16, mizuNamespace string, mizuServiceName string) string {
-	return fmt.Sprintf("localhost:%d/api/v1/namespaces/%s/services/%s:80/proxy", mizuPort, mizuNamespace, mizuServiceName)
+const mizuServicePort = 80
+
+func getMizuCollectorProxiedHostAndPath(mizuNamespace string, mizuServiceName string) string {
+	return fmt.Sprintf("/api/v1/namespaces/%s/services/%s:%v/proxy/", mizuNamespace, mizuServiceName, mizuServicePort)
```

Comment:
Use `%d` here instead of `%v`, because this is not just a log message. It is important that we print an int here.
If for some reason `mizuServicePort` changes to an object, then we will notice it.